node-module-system/calculate.js

```diff
@@ -0,0 +1,34 @@
+import add from './add.js';
+import multiply from './multiply.js';
+import subtract from './subtract.js';
+import divide from './divide.js';
+
+switch (process.argv[3]) {
+  case 'plus':
+    console.log(
+      'Result:',
+      add(Number(process.argv[2]), Number(process.argv[4]))
```

Comment:
You convert to Number over and over again. It would be nice to do it once.

Reply:
Updated